# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/target/lvp/patches/linux24_quiet.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- linux-2.4.21_orig/init/main.c	2003-07-25 16:13:42.000000000 +0200
+++ linux-2.4.21/init/main.c	2003-07-25 16:16:30.000000000 +0200
@@ -210,7 +210,7 @@ static int __init quiet_kernel(char *str
 {
 	if (*str)
 		return 0;
-	console_loglevel = 4;
+	console_loglevel = 3;
 	return 1;
 }
 
